exec/java-exec/src/test/java/org/apache/drill/exec/physical/unit/PhysicalOpUnitTestBase.java

```diff
@@ -82,6 +81,8 @@
  * Look! Doesn't extend BaseTestQuery!!
  */
 public class PhysicalOpUnitTestBase extends ExecTest {
+  public static long INIT_ALLOCATION = 10_000_000l;
+  public static long MAX_ALLOCATION = 15_000_000L;
```

Comment:
Should we use the values already defined in `AbstractBase`? To do that, perhaps refactor the code from:

```
public abstract class AbstractBase implements PhysicalOperator{
  ...
  protected long initialAllocation = 1_000_000L;
  protected long maxAllocation = 10_000_000_000L;
```

To:

```
public abstract class AbstractBase implements PhysicalOperator{
  ...
  public static long INIT_ALLOCATION = 1_000_000L;
  public static long MAX_ALLOCATION = 10_000_000_000L;
  protected long initialAllocation = INIT_ALLOCATION;
  protected long maxAllocation = MAX_ALLOCATION;
```

Actually, I think I did that in my much-delayed external sort PR for unit testing...